update.sh

```diff
@@ -50,14 +53,14 @@ for variant in "${variants[@]}"; do
   fi
   maintainer=${maintainers[$variant]:-${maintainers[$bv]}}
   { generated_warning; cat "$template"; } > "$dir/Dockerfile"
-  ( sed -i '' 's!%%BASE_TAG%%!'"$openjdk_version"'!g' "$dir/Dockerfile"
-    sed -i '' 's!%%MAINTAINER%%!'"$maintainer"'!g' "$dir/Dockerfile"
-    sed -i '' 's!%%BUILD_TOOL_VERSION%%!'"$bt_version"'!g' "$dir/Dockerfile"
+  ( sed -i 's!%%BASE_TAG%%!'"$openjdk_version"'!g' "$dir/Dockerfile"
+    sed -i 's!%%MAINTAINER%%!'"$maintainer"'!g' "$dir/Dockerfile"
+    sed -i 's!%%BUILD_TOOL_VERSION%%!'"$bt_version"'!g' "$dir/Dockerfile"
     if [ "$bv" = "alpine" ]; then
-      sed -i '' 's/^%%ALPINE%% //g' "$dir/Dockerfile"
+      sed -i 's/^%%ALPINE%% //g' "$dir/Dockerfile"
     else
-      sed -i '' '/^%%ALPINE%%/d' "$dir/Dockerfile"
-      sed -i '' '/^$/N;/^\n$/D' "$dir/Dockerfile"
+      sed -i '/^%%ALPINE%%/d' "$dir/Dockerfile"
+      sed -i '/^$/N;/^\n$/D' "$dir/Dockerfile"
```

Comment:
So this was breaking GNU sed the way it was before, I assume? Now it breaks whatever BSD version of sed macOS ships with. 🙄

We'll need to find a way to make it work with both. Since I introduced the macOS exclusivity, I'm happy to take a crack at it. I think last time I ran into something similar I just switched to perl. 😁